.github/workflows/release.yaml

```
@@ -45,6 +46,7 @@ jobs:
 
   release-charts-to-acr:
     runs-on: ubuntu-latest
+    if: ${{ github.repository == 'GreptimeTeam/helm-charts' }}
     if: ${{ inputs.release-charts-to-acr == 'true' || github.event_name == 'push' }}
```

actionlint (1.7.4)

key "if" is duplicated in "release-charts-to-acr" job. previously defined at line:49,col:5

(syntax-check)